content/sensu-go/6.7/operations/control-access/namespaces.md

```diff
@@ -64,6 +64,10 @@ To organize resources, use labels rather than multiple namespaces.
 Every [Sensu backend][1] includes a `default` namespace.
 All resources created without a specified namespace are created within the `default` namespace.
 
+At startup, Sensu also creates the `sensu-system` namespace to contain [backend entities][12].
+The `sensu-system` namespace and backend entities facilitate backend error reporting and make Sensu deployments more observable by enabling backend-generated status and metrics events.
+Only cluster admins have access to the `sensu-system` namespace.
+
```

Comment:
I should probably add something like `If you have cluster admin permissions, you can use sensuctl and the web UI to access backend entities like other entities.` here too

Reply:
Correct.